app/models/concerns/justifiable.rb

```diff
@@ -5,16 +5,24 @@ module Justifiable
     attr_accessor :ticket_number
     attr_reader :ticket_url
 
+    before_validation :strip_octothorpe
     validates :ticket_number,
               presence: true,
-              numericality: { only_integer: true },
+              numericality: { only_integer: true,
+                              message: :numeric_with_optional_hash },
```

Comment:
i18n-tasks doesn't recognize the Active Record error message keys, so this will make it stop complaining.
```suggestion
              # i18n-tasks-use t("activerecord.errors.messages.numeric_with_optional_hash")
              numericality: { only_integer: true, message: :numeric_with_optional_hash },
```